atsign-foundation/at_server
# additional stats

# What I did #

Added new Notification stats. Have done some modifications in other stats too. Here are sample stats [output](https://cdn.sourceb.in/bins/c0vX4PZJoA/0).

### Modified

> stats:5

**Old :** `{"@minnu": "1"}`

**New :** `{@minnu: 1}`

> stats:6

**Old :** `{"signing_publickey@sunny": "4"}`

**New :** `{signing_publickey@sunny: 4}`

> stats:9 (diskSize)

**Old :** `"25.5 KB"`

**New :** `{size: 8.54, units: KB}`

# How I did it #

- Forked the code and updated the code.

# How to verify it #
- run the root server and launch the secondary and authenticate.
- Use stats:11 to see new stats

# Description for the changelog #
